buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1449,7 +1454,8 @@ class BeamModulePlugin implements Plugin<Project> {
         // Note that these tests will fail on JVMs that JMH doesn't support.
         def jmhTest = project.tasks.register("jmhTest", JavaExec) {
           dependsOn project.classes
-          mainClass = "org.openjdk.jmh.Main"
+          // Note: this will delegate to the default JMH runner
```

Comment:
```suggestion
          // Note: this will wrap the default JMH runner publishing results to InfluxDB
```

Reply:
For `jmhTest` the code just delegates to the default JMH runner, we don't want to publish single shot times. I'll updated to comment accordingly